### PR TITLE
Enhance UI on Initial Page

### PR DESCRIPTION
**Pull request by mla04762**

### CODE REVIEW

**Comment by sharonpatta** [comment]
Thanks for creating a component for the app metadata. Could you possibly move it to a new /components folder right under src?

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+        />
+    </figure>
+    <div class="card-body">
+        <h2 class="card-title">Shoes!</h2>
+        <p>If a dog chews shoes whose shoes does he choose?</p>
+        <div class="card-actions justify-end">
+            <button class="btn btn-primary">Buy Now</button>
+        </div>
+    </div>
+</div>
```
Comment (jaeminlee0515) [inline src/routes/+page.svelte]: Could you revert back to the original page? It seems the shoe card that pops up was an example from when you were testing it out.

**Review by jaeminlee0515 — CHANGES_REQUESTED** [review]

File: src/routes/+page.svelte (hunk 1 above)
Comment (mla04762) [reply src/routes/+page.svelte]: Re-pushed after removing this code

File: src/routes/app/[id]/+page.svelte [hunk 2]
```diff
@@ -4,30 +4,35 @@
 <script lang="ts">
     //	import Post from '$lib/components/Post.svelte';
     import type { PageData } from './$types';
+    import AppMetadata from '../../../components/AppMetadata.svelte';
```
Comment (chrisvire) [inline src/routes/app/[id]/+page.svelte]: Use `$lib/src/components/AppMetadata.svelte`

File: src/components/AppMetadata.svelte [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<!-- AppDetails.svelte -->
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+
+    import type { PageData } from '../routes/app/[id]/$types';
```
Comment (chrisvire) [inline src/components/AppMetadata.svelte]: This is an inappropriate coupling. It is more appropriate to have separate `export let` properties.  

Also, how will you deal with selecting the correct localization based on the browser language?  Currently, it is hardcoded to `listings[0]`. We can wait on that for a little longer, but it should be scheduled.

**Review by chrisvire — CHANGES_REQUESTED** [review]

**Comment by chrisvire** [comment]
@mla04762 Please put a screenshot of a real app in the comments here.

**Comment by mla04762** [comment]
Collapsed Long Description:
Screenshot 2024-02-06 at 10 34 59 AM

Expanded Long Description:
Screenshot 2024-02-06 at 10 35 09 AM

**Review by chrisvire — APPROVED** [review]
I will approve the technical aspects of this PR.  There is more work that needs to be done on the user experience.

**Pull request merged**